remotes/docker/schema1/converter.go

```diff
@@ -321,6 +324,7 @@ func (c *Converter) fetchBlob(ctx context.Context, desc ocispec.Descriptor) erro
 		})
 
 		if err := eg.Wait(); err != nil {
+			log.G(ctx).Errorf("fetch blob %s error %v", desc.Digest, err)
```

Comment:
Does the caller not properly handle or log this error?

Reply:
you are right, the caller will handle this error, I'll get rid of this log